modules/javafx.graphics/src/main/java/javafx/css/PseudoClass.java

```diff
@@ -74,6 +74,12 @@
  */
 public abstract class PseudoClass {
 
+    /**
+     * Creates a default {@code PseudoClass}.
+     */
+    public PseudoClass() {
+    }
+
     /**
```

Comment:
1. Is having a public constructor for this class a good idea? Instances are created by a factory method.
2. Both methods in this class need documentation update. `getPseudoClass` has a poor method description and the formatting of the `@return` tag is wrong (lowercase and no period). `getPseudoClassName` is missing a description.

Reply:
Maybe we can add the method docs fixes to [JDK-8250590](https://bugs.openjdk.java.net/browse/JDK-8250590)?